model/dao/TaskDAO/PostgreSQLTaskDAO.php

```diff
@@ -968,7 +921,7 @@ public function update(TaskVO $taskVO) {
     public function create(TaskVO $taskVO) {
         $affectedRows = 0;
 
-        $sql = "INSERT INTO task (_date, init, _end, story, telework, onsite, text, ttype, phase, usrid, projectid, task_storyid) VALUES(" .
+        $sql = "INSERT INTO task (_date, init, _end, story, telework, onsite, text, ttype, phase, usrid, projectid) VALUES(" .
```

Comment:
What's the difference between story and task story id?

Reply:
In the "Coordination" menu, you used to be able to create iterations and stories inside of them, which contained tasks (named task story). Tasks could be assigned to a task inside a story through the task story id field. The tasks UI was able to list the open stories for a certain project to select among them.

Story is a text identifier that was originally used to relate tasks with stories in an external coordination tool. It's still useful as a free-form identifier to classify tasks (there's a report for that), that's why I'm keeping this one and not the other.

Reply:
Got it, thank you